types/guid/index.d.ts

```diff
@@ -0,0 +1,37 @@
+// Type definitions for guid 1.0
+// Project: https://github.com/dandean/guid
+// Definitions by: Marc-Andre Roy <https://github.com/maroy1986>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+export = guid;
+
+declare function guid(guid: object): object;
+
+declare namespace guid {
+    const EMPTY: string;
+
+    const prototype: {
+    };
+
+    function create(): object;
+
+    function isGuid(value: string): boolean;
+
+    function raw(): string;
+
+    namespace create {
```

Comment:
why do you have namepsaces for the functions? they are not needed ?

Reply:
You have a good point, those are not required after verification, they just get generated by `dts-gen`. I'll get them removed.